config/initializers/devise.rb

```diff
@@ -265,7 +265,23 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+
+  scheme = Rails.application.config.force_ssl ? "https" : "http"
+  if Rails.application.config.manyfold_features[:oidc]
+    config.omniauth :openid_connect, {
+      name: :openid_connect,
+      issuer: "https://#{ENV["OIDC_PROVIDER_HOST"]}/",
+      scope: [:openid, :email, :preferred_username, :nickname],
```

Comment:
The default OIDC scope for a nickname field would be `profile`.

Reply:
yeah, I was trying to cut down the asked-for info as much as possible, but maybe just `:profile` is OK.